civihr_employee_portal/js/reports.js

```diff
@@ -331,13 +330,57 @@
     }
 
     /**
+     * Init angular in reports custom page,
+     * and add the calendar icon
+     *
+     */
+    HRReport.prototype.initAngular = function() {
+        var self = this;
+
+        require([
+            'common/angular',
+            'common/services/angular-date/date-format',
+            'common/directives/angular-date/date-input',
+            'common/moment',
+            'common/filters/angular-date/format-date'
+        ], function() {
+            angular.module('hrr', [
+                'ngAnimate',
+                'ui.bootstrap',
+                'common.angularDate'
+            ]).controller('FiltersController', function($scope, $timeout) {
+                $scope.format = 'dd/MM/yyyy';
+
+                $('[ng-model]').each(function () {
+                    $scope[$(this).attr('is-open')] = false;
+                });
+
+                $scope.open = function(id) {
+                    if ($('#bootstrap-theme').length === 0) {
+                        $('.datepicker-popup').wrap('<span id="bootstrap-theme" />');
```

Comment:
Why adding the `bootstrap-theme` namespace via js? Can't you add it to the html itself? (And again, you shouldn't do any DOM manipulation in the controller)